labs/ElectrodoDoc.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: ElectdoroDoc
+[//]: Electrodo # (parent: Electrodo)
+---
+# Lab Name
+ElectdoroDoc
+
+# Short Description
+Electrodo - software service for climate-related assets and risk management to increase enterprise market value. Electrodo solves the problem of data/context integrity, data/context reliability, and trust for carbon-related assets lifecycle, climate-related risk management, and ESG reporting by using a standardized framework based on TCFD Recommendations and data verification on the blockchain.
+# Scope of Lab
+During the Prototype Challenge, we developed a prototype ElectrodoDoc (Fig. 1) that implements the basic functionality for the climate-related disclosure service based on Hyperledger Fabric, FIWARE context broker (Orion), and IPFS protocol. Our solution demonstrates ESG data/context management and self-verification principles in terms of climate-related disclosures' lifecycle: data/context collection and processing, data/context verification, and publication of sustainability report in line with TCFD Recommendations framework.
+![Fig 1.](https://user-images.githubusercontent.com/17604217/170219272-5d6c62ff-a3a0-4cac-940a-6959fe486981.png)
+
+# Initial Committers
+_Enter the Github IDs for the set of initial committers._
+- https://github.com/bismerk
+- https://github.com/andrii482
+- https://github.com/alexshehovcov
+- https://github.com/Dmitrii-Kholodov
+- https://github.com/roman-kravchenko
+
+# Sponsor
+
+
+# Pre-existing repository
+- https://github.com/482solutions/ElectrodoDoc-meta
```

Comment:
Please note that any repository that we transfer into Hyperledger Labs must meet the following criteria:
1. Be Apache 2.0 licensed
2. Have DCO sign-off for each of the commits. 

In looking at this repo, I see that neither of these are met. These will need to be fixed before we could transfer this repo to the hyperledger-labs organization.

Reply:
@tkuhrt
Hello. Is it okay, if DCO sign-off would be only in 'ElectrodoDoc-meta' repository (but not exist in other mentioned  in 'ElectrodoDoc-meta' repository repositories) ?

Reply:
I am not sure that I understand the question. There is a single repository listed above and as it currently stands, not all of the commits in this repository have DCO sign-offs. Before we can transfer this repo to the hyperledger-labs organization, we would need to have DCO sign-off completed.

Reply:
I see that the commits have been changed and a license file added. Ignore my previous comment.